Link errors on OSX and Linux [build 04a69b7]

Version  04a69b7  will not compile on Fedora 23.  I have an automated script that runs and I am now receiving an error....

[100%] Linking CXX executable ../../freeorion
CMakeFiles/freeorion.dir/**/**/UI/ObjectListWnd.cpp.o: In function `boost::detail::function::function_obj_invoker3<(anonymous namespace)::CustomRowCmp, bool, GG::ListBox::Row const&, GG::ListBox::Row const&, unsigned long>::invoke(boost::detail::function::function_buffer&, GG::ListBox::Row const&, GG::ListBox::Row const&, unsigned long)':
ObjectListWnd.cpp:(.text+0x2918): undefined reference to`boost::locale::generator::generate(std::**cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
ObjectListWnd.cpp:(.text+0x2975): undefined reference to `boost::locale::generator::generator()'
ObjectListWnd.cpp:(.text+0x298e): undefined reference to`boost::locale::generator::~generator()'
collect2: error: ld returned 1 exit status
client/human/CMakeFiles/freeorion.dir/build.make:1848: recipe for target 'freeorion' failed
make[2]: **\* [freeorion] Error 1
CMakeFiles/Makefile2:454: recipe for target 'client/human/CMakeFiles/freeorion.dir/all' failed
make[1]: **\* [client/human/CMakeFiles/freeorion.dir/all] Error 2
Makefile:149: recipe for target 'all' failed
make: **\* [all] Error 2
Install Free Orion
Building v0.4.5+ build 2016-02-07.04a69b7
[  0%] Built target freeorionversion
[ 24%] Built target freeorioncommon
[ 40%] Built target GiGi
[ 41%] Built target GiGiSDL
[ 61%] Built target freeorionparse
[ 67%] Built target freeoriond
[ 73%] Built target freeorionca
[ 73%] Linking CXX executable ../../freeorion
CMakeFiles/freeorion.dir/**/__/UI/ObjectListWnd.cpp.o: In function `boost::detail::function::function_obj_invoker3<(anonymous namespace)::CustomRowCmp, bool, GG::ListBox::Row const&, GG::ListBox::Row const&, unsigned long>::invoke(boost::detail::function::function_buffer&, GG::ListBox::Row const&, GG::ListBox::Row const&, unsigned long)':
ObjectListWnd.cpp:(.text+0x2918): undefined reference to`boost::locale::generator::generate(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
ObjectListWnd.cpp:(.text+0x2975): undefined reference to `boost::locale::generator::generator()'
ObjectListWnd.cpp:(.text+0x298e): undefined reference to`boost::locale::generator::~generator()'
collect2: error: ld returned 1 exit status
client/human/CMakeFiles/freeorion.dir/build.make:1848: recipe for target 'freeorion' failed
make[2]: **\* [freeorion] Error 1
CMakeFiles/Makefile2:454: recipe for target 'client/human/CMakeFiles/freeorion.dir/all' failed
make[1]: **\* [client/human/CMakeFiles/freeorion.dir/all] Error 2
Makefile:149: recipe for target 'all' failed
make: **\* [all] Error 2

Thanks.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -125,7 +125,7 @@ set(LIB_SUFFIX ${OLD_LIB_SUFFIX})
 ########################################
 # Dependencies                         #
 ########################################
-find_package(Boost ${MINIMUM_BOOST_VERSION} COMPONENTS chrono date_time filesystem iostreams regex serialization signals system thread log REQUIRED)
+find_package(Boost ${MINIMUM_BOOST_VERSION} COMPONENTS chrono date_time filesystem locale iostreams regex serialization signals system thread log REQUIRED)
 find_package(ZLIB REQUIRED)
 find_package(Freetype REQUIRED)
 

